src/less/breadcrumb/ds6/breadcrumb-base.less

```diff
@@ -0,0 +1,35 @@
+.breadcrumb {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    list-style: none;
+    margin: 0;
+    li {
+        display: inline-block;
+        a {
+            color: #000000;
```

Comment:
Please use `color: @ds6-color-text-default;`

Reply:
@ds6-color-text-default is #111820;. as per mock, it should be #000000

Reply:
Hmmm. I haven't seen direct black as part of the standard. Can we confirm with the designer(s)?

Reply:
Per our email and Slack conversations, this should be the variable I gave you. 👍